ui/src/components/table/table-element/SortArrow.tsx

```diff
@@ -0,0 +1,17 @@
+import { ChevronUp, ChevronDown } from 'lucide-react';
+
+interface SortArrowProps {
+    direction: string | boolean;
+}
+
+const SortArrow = ({ direction }: SortArrowProps) => (
```

Comment:
Same here as I mentioned in GroupingsTable.tsx.

Reply:
done